pom.xml

```diff
@@ -67,6 +67,8 @@ THE SOFTWARE.
     <project.build.outputEncoding>UTF-8</project.build.outputEncoding>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <findbugs.failOnError>false</findbugs.failOnError>
+    <!-- TODO: Required to override by a version with certchain support (MJARSIGNER-53) on @oleg-nenashev's machine. Remove once it's in upstream -->
```

Comment:
Where is “upstream”? Link to the PR where you are adding this.

Reply:
Links are provided in the original pull-request: https://github.com/jenkinsci/remoting/pull/190